[Email Security] Enable auto-move events for manually added domains (#17960)
cloudflare/cloudflare-docs: src/content/docs/cloudflare-one/email-security/setup/post-delivery-deployment/bcc-journaling/journaling-setup/manual-add.mdx

## Enable auto-moves

To enable auto-move events, you will have to connect and associate an integration:

1. Go to **Settings** > **Domain management** > **Integrated domains** > Select **View**.
2. On the **Domains** page, select the three dots, then select **Associate an integration**.
3. Select **Connect an integration**. Follow the steps to [enable Microsoft integration](/cloudflare-one/email-security/setup/post-delivery-deployment/bcc-journaling/journaling-setup/office365-journaling/#integrate-with-microsoft-o365). This step will allow you to create an integration.
4. Once you completed the steps to enable Microsoft integration, select **Continue to Email Security**. You will now be redirected to the **Domains** page, where you will be able to connect the integration to your specified domain.

> 5. Select the three dots, then select **Associate an integration**. Select the integration, then select **Associate**. This step will allow you to associate an integration to your selected domain.

Reviewer: ```suggestion
5. Select the three dots, then select **Associate an integration**. Select the integration, then select **Associate**. This step associates an integration to your selected domain.
```
Nit: Consider removing the last sentence (like in step 3).

> 6. Now that your domain has an associated integration, you can enable [auto-move events](/cloudflare-one/email-security/auto-moves/) on your domain.

Reviewer: ```suggestion
6. Now that your domain has an associated integration, enable [auto-move events](/cloudflare-one/email-security/auto-moves/) on your domain.
```
